platform/core/src/capabilities/InstantiationCapability.js

```diff
@@ -45,19 +48,26 @@ define(
          * have been persisted, nor will it have been added to the
          * composition of the object which exposed this capability.
          *
+         * @param {object} the model for the new domain object
          * @returns {DomainObject} the new domain object
          */
         InstantiationCapability.prototype.instantiate = function (model) {
+            var parsedId =
+                    this.identifierService.parse(this.domainObject.getId()),
+                space = parsedId.getDefinedSpace(),
```

Comment:
This behavior is the only usage of getDefinedSpace; is there any reason why using `getSpace` wouldn't suffice?  I don't see why we wouldn't want to use the default space when one isn't provided.  A space has to be chosen at some point.